README.md

```diff
@@ -346,6 +440,11 @@ This provider allows raw configurations to be managed by Puppet. It serves as a
 
 Manages AAA Authentication Login configuration.
```

Comment:
**General Comment** In each minimum requirements table (per provider) we list versions even if they are not currently supported on that platform.  We should add unsupported instead.